module/zfs/vdev_raidz.c

```diff
@@ -45,6 +45,12 @@
 
 #ifdef ZFS_DEBUG
 #include <sys/vdev.h>	/* For vdev_xlate() in vdev_raidz_io_verify() */
+
+#ifndef _KERNEL
+#include <zlib.h>
+#else
+#include <linux/crc32.h>
```

Comment:
Presumably need a portable way to do this

Reply:
This PR is not need to be included in https://github.com/openzfs/zfs/pull/12225.
It is just debug logic to trace raidz blocks corruptions.

Reply:
I'd say it can not be included.  It is too verbose and expensive for just ZFS_DEBUG.  While it may be useful, don't confuse Matt more than needed, or this project will never end. ;)